upload/admin/view/template/module/smaily_for_opencart.tpl

```diff
@@ -321,6 +317,74 @@
                 </div>
               </div>
             </div>
+            <!-- RSS -->
+            <div id="section4" class="tab-pane fade in">
+              <div class="form-group">
+                <label class="col-sm-2 control-label" for="rss-category"><?php echo $rss_category_title; ?></label>
+                <div class="col-sm-10">
+                  <select name="smaily_for_opencart_rss_category" id="rss-category" class="form-control smaily-rss-options">
+                    <option value="">All Products</option>
+                    <?php
+                    foreach ($rss_categories as $category) {
+                      $selected = $category['category_id'] === $rss_category ? 'selected' : ''; ?>
+                      <option value="<?php echo $category['category_id']; ?>"<?php echo $selected; ?>><?php echo $category['name']; ?></option>
+                    <?php } ?>
+                    </select>
+                </div>
+              </div>
+              <div class="form-group">
+                <label class="col-sm-2 control-label" for="rss-sort-by"><?php echo $rss_sort_by_title; ?></label>
+                <div class="col-sm-10">
+                  <select name="smaily_for_opencart_rss_sort_by" id="rss-sort-by" class="form-control smaily-rss-options">
+                   <?php
+                    // Add options for select.
+                    foreach ($sort_options as $sort_code => $sort_name) {
+                      $selected = $rss_sort_by === $sort_code ? 'selected' : ''; ?>
+                      <option value="<?php echo $sort_code; ?>"<?php echo $selected; ?>><?php echo $sort_name; ?></option>
```

Comment:
Missing whitespace between `value` and `selected` attributes.

Reply:
This is fixed